theme.go

```diff
@@ -48,6 +48,8 @@ type FieldStyles struct {
 	// Select styles.
 	SelectSelector lipgloss.Style // Selection indicator
 	Option         lipgloss.Style // Select options
+	SelectNext     lipgloss.Style
+	SelectPrev     lipgloss.Style
```

Comment:
@meowgorithm, not 100% sure on these names. Do you have any thoughts?

Maybe something more generic such as `NextIndicator` `BackIndicator`?

Reply:
Good call. How about `NextIndicator` and `PrevIndicator`?

Reply:
Yup, sounds good to me!